README.md

```diff
@@ -2,48 +2,75 @@
+
+
+#### Without Macaroons (insecure)
+
+If you don't need to access the `tdexd` from outside over the public internet, it's enough to [uncomment this line to set TDEX_NO_MACAROONS=true](docker-compose.yml#L18) env var of `tdexd` to disable it. If you do this, you are done and you can skip to the [Run](#run) section. **DO NOT OPEN THE OPERATOR GRPC PORT VIA FIREWALL/SECURITY GROUPS**
+
+
 
-### TLS
+#### With Macaroons & TLS self-signed
 
-#### Trade interface
+The TLS key and certificate are generated by the `tdexd` at first startup. 
+If you want to run the daemon on a remote machine with a static IP (and possibly a DNS name), you need to uncomment one or both at the same time [TDEX_OPERATOR_EXTRA IP | TDEX_OPERATOR_EXTRA_DOMAIN](docker-compose.yml#L21) env vars to let the daemon create the right certificate.
 
-To enable TLS encryption on the Trade interface of the daemon, you have to generate your own TLS key and certificate files. Once this is done, export the ENV variables with the absolute path of the files: 
+
+To let the `feederd` to connect to `tdexd` to update prices, you'll need to change the default `feederd/config.json` file like follows:
+
+- change the [hostname of the rpc_address](feederd/config.json#L13) and use the static IP or DNS name you passed to `TDEX_OPERATOR_EXTRA IP` or  `TDEX_OPERATOR_EXTRA_DOMAIN` eg. `my.domain.com:9000`
+- uncomment the [feederd's volumes](docker-compose.yml#54) section in the compose file to mount the macaroon and TLS certificate intto `feederd` service
+- change the [macaroons_path](feederd/config.json#L11) and [tls_cert_path](feederd/config.json#L12) in the `feederd/config.json` respectively to `/price.macaroon` and `/cert.pem`.
+- Last but not least, don't forget to open the port where the Operator interface listens to (default `9000`) and allow in-going and out-going traffic over it on your firewalle or security groups.
+
+
+# 5. Trader interface  
+Expose the TDEX Trader interface with TLS or Onion to the TDEX Network
+
+### TLS via Certificate Authority
+
+To enable TLS encryption with a Certificate Authority on the Trader interface of the daemon, you have to generate your obtain a TLS certificate from a Certificate Authority (ie. Let's Encrypt or ZeroSSL). Once this is done, export the ENV variables with the absolute path of the files: 
 
 ```sh
 $ export SSL_CERT_PATH=/path/to/fullchain.pem
 $ export SSL_KEY_PATH=/path/to/privatekey.pem
 ```
 
-The last step is to uncomment in the compose file (`docker-compose.yml`) the lines related to TLS for Trade interface.
-
-#### Operator interface
-
-By default, the daemon service is configured to enable TLS encryption for the Operator interface. It's enough to uncomment [TDEX_NO_MACAROONS](docker-compose.yml#L18) env var to disable it.
+The last step is to uncomment in the compose file (`docker-compose.yml`) the lines of the `volumes` to mount the SSL paths
```

Comment:
```suggestion
The last step is to uncomment in the compose file [the lines](docker-compose#L39) to mount these paths as volumes of `tdexd` in the paths `/trade.crt` and `/trade.key`.
```

We are actually missing to describe that also [these lines](https://github.com/tdex-network/tdex-box/blob/master/docker-compose.yml#L26) must be uncommented.